realm/functions/send_status_to_slack/source.js

```diff
@@ -0,0 +1,21 @@
+exports = async function (isError, message) {
```

Comment:
Great idea to extract this out into a function. I don't know why it didn't occur to me that you'd do this.